hack/generate-manifest.sh

```diff
@@ -24,6 +24,7 @@ _usage="Usage: $0 [--mode (dev|release)] [--keep] [--help|-h]
 Generate a YAML manifest for the Clickhouse-Grafana Flow-visibility Solution, using Helm and
 Kustomize, and print it to stdout.
         --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
+        --spark-operator      Generate a manifest with spark-operator support enabled
```

Comment:
I feel it would be better to highlight that only the Deployment and Service Account will be included in the generated yaml. The crd part will not be included/need to separately apply spark-operator-crds.yaml etc.

Reply:
Thanks. I added some descriptions, but still feel it might be strange as we do not mention that the ClickHouse CRDs are required before applying flow-visibility in this script.

Actually, I'm considering about injecting the spark-operator-crds.yaml into flow-visibility.yaml with Kustomize in this script when using `--spark-operator` option. As the installation order does not matter, this might make the deployment easier. Would like to hear some suggestions.

Reply:
> I'm considering about injecting the spark-operator-crds.yaml into flow-visibility.yaml with Kustomize in this script when using --spark-operator option.

I think it's a good idea

Reply:
> I'm considering about injecting the spark-operator-crds.yaml into flow-visibility.yaml with Kustomize in this script when using --spark-operator option.

Sounds good to me.

> but still feel it might be strange as we do not mention that the ClickHouse CRDs are required before applying flow-visibility in this script.

Similar cases with the above one, since we already included the ClickHouse deployment in `flow-visibility.yaml`, what about include ClickHouse CRDs inside `flow-visibility.yaml`?

Reply:
We cannot include ClickHouse CRDs inside `flow-visibility.yaml`, as ClickHouse CRDs must be applied before we use the Kind `ClickHouseInstallation`.